src/modules/Todo.js

```diff
@@ -0,0 +1,68 @@
+const descriptionInput = document.getElementById('description-input');
+let allToDos = [];
+const todosList = document.getElementById('todos');
+
+const deleteToDoItem = (index) => {
+  allToDos.splice(index, 1);
+  todosList.innerHTML = '';
+  localStorage.setItem('alltasks', JSON.stringify(allToDos));
+};
+
+const editToDoList = (index, newValue) => {
+  allToDos[index].description = newValue;
+};
+
+const toggleCompletedTask = (index) => {
+  allToDos[index].completed = !allToDos[index].completed;
+};
```

Comment:
- [ ]  As previous reviewer suggested, you still have some tasks that are true by default. while they should be true only if the user check the tasks as done. 
![image](https://user-images.githubusercontent.com/43877689/228908327-1790f794-b7fe-4275-809b-e1cdb6a37f69.png)